data/json/monsters/insect_spider.json

```diff
@@ -383,6 +383,71 @@
     "delete": { "flags": [ "SMALL_HIDER" ] },
     "special_attacks": [ [ "EAT_FOOD", 40 ], [ "EAT_CARRION", 40 ] ]
   },
+  {
+    "id": "mon_silverfish_small",
+    "type": "MONSTER",
+    "name": { "str_sp": "oversized silverfish" },
+    "description": "An oversized mutated silverfish the size of a small dog.",
+    "copy-from": "mon_silverfish",
+    "proportional": { "armor": 0.45, "speed": 1.1, "morale": 0.67, "melee_dice_sides": 0.67 },
+    "volume": "15 L",
+    "weight": "15 kg",
+    "hp": 23,
+    "zombify_into": "mon_meat_cocoon_tiny",
+    "upgrades": { "age_grow": 30, "into": "mon_giant_cockroach" },
+    "extend": { "flags": [ "NO_BREED" ] }
+  },
+  {
+    "id": "mon_silverfish",
+    "type": "MONSTER",
+    "name": { "str_sp": "giant silverfish" },
+    "description": "A mutant silverfish roughly the size of a person.  Doesn't seem at all interested in you.",
+    "default_faction": "insect",
+    "bodytype": "insect",
+    "species": [ "INSECT" ],
+    "volume": "60 L",
+    "weight": "60 kg",
+    "hp": 71,
+    "speed": 135,
+    "material": [ "iflesh" ],
+    "symbol": "s",
+    "color": "dark_gray",
+    "aggression": -20,
+    "morale": 60,
+    "stomach_size": 250,
+    "melee_skill": 4,
+    "melee_dice": 1,
+    "melee_dice_sides": 4,
+    "melee_damage": [ { "damage_type": "cut", "amount": 3 } ],
+    "dodge": 4,
+    "families": [ "prof_gross_anatomy", "prof_intro_biology", "prof_physiology", "prof_wp_basic_bug" ],
+    "weakpoint_sets": [ "wps_arthropod" ],
+    "vision_day": 5,
+    "vision_night": 30,
+    "harvest": "arachnid",
+    "dissect": "dissect_insect_sample_single",
+    "reproduction": { "baby_egg": "egg_silverfish", "baby_count": 3, "baby_timer": 15 },
+    "baby_flags": [ "SPRING", "SUMMER", "AUTUMN", "WINTER" ],
+    "special_attacks": [ [ "EAT_FOOD", 60 ], [ "EAT_CROP", 60 ] ],
+    "zombify_into": "mon_meat_cocoon_small",
+    "fear_triggers": [ "BRIGHT_LIGHT", "HURT", "FRIEND_ATTACKED", "FRIEND_DIED" ],
+    "flags": [ "SEES", "HEARS", "SMELLS", "CLIMBS", "EATS", "PATH_AVOID_DANGER", "SMALL_HIDER" ],
+    "armor": { "bash": 3, "cut": 5, "bullet": 1 }
+  },
+  {
+    "id": "mon_silverfish_nymph",
+    "type": "MONSTER",
+    "name": { "str": "giant silverfish nymph" },
+    "description": "A baby mutant silverfish about the size of a rat.",
+    "copy-from": "mon_larva_abstract",
+    "default_faction": "insect",
+    "color": "dark_gray",
+    "stomach_size": 80,
+    "fear_triggers": [ "BRIGHT_LIGHT" ],
+    "upgrades": { "age_grow": 30, "into": "mon_silverfish" },
```

Comment:
```suggestion
    "upgrades": { "age_grow": 30, "into": "mon_silverfish_small" },
```

Reply:
No, this is intended. None of our larvae grow into the oversized versions, because the oversized versions don't reproduce. They grow to giant.